src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -608,5 +611,19 @@ public async Task<bool> RestartWorkerWithInvocationIdAsync(string invocationId)
             }
             return false;
         }
+
+        internal void CheckPackageJson()
+        {
+            if (_workerRuntime == RpcWorkerConstants.NodeLanguageWorkerName && _environment.AzureFilesAppSettingsExist())
```

Comment:
Add a comment of our findings and why cold start could be affected if package.json is missing?